src/DropdownButton.js

```diff
@@ -1,50 +1,46 @@
 import React from 'react';
-import classNames from 'classnames';
-import uncontrollable from 'uncontrollable';
 import BootstrapMixin from './BootstrapMixin';
-import ButtonGroup from './ButtonGroup';
-import DropdownBase from './DropdownBase';
-import DropdownToggle from './DropdownToggle';
-import DropdownMenu from './DropdownMenu';
+import Dropdown from './Dropdown';
 import NavDropdown from './NavDropdown';
 import CustomPropTypes from './utils/CustomPropTypes';
 import deprecationWarning from './utils/deprecationWarning';
+import omit from 'lodash/object/omit';
+
+class DropdownButton extends React.Component {
 
-class DropdownButton extends DropdownBase {
   constructor(props) {
     super(props);
   }
 
   render() {
-    if (this.props.navItem) {
-      return <NavDropdown {...this.props} />;
-    }
```

Comment:
I didn't realize that I hadn't added a deprecation warning inside that If block. May still be worth including?